deps/llvm.mk

```diff
@@ -430,6 +430,7 @@ $(eval $(call LLVM_PATCH,llvm-windows-race))
 endif
 $(eval $(call LLVM_PATCH,llvm-D51842-win64-byval-cc))
 $(eval $(call LLVM_PATCH,llvm-D57118-powerpc))
+$(eval $(call LLVM_PATCH,llvm-r355582-avxminmax))
```

Comment:
This needs a `# remove for Y.0` comment since this commit is on master (Y being whatever major LLVM version will be the first to have this patch).

Reply:
Added a comment with 8.0, but how do I make sure of that? I checked the rc branches on the LLVM Phabricator but it hasn't been added to any of them yet.